topic_name: /scan
queue_size: 1
gain: 100.0
sv_name: e_switch